wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -718,7 +722,96 @@ public boolean uploadFileLocalDropZone(File file) throws Exception
 
         return uploadFile(file, fetchLocalDropZones[0]);
     }
+    
+    /**
+     * UPLOADS A FILE TO THE SPECIFIED LANDING ZONE
+     * USED BY MINIAPPS FOR UP TO 2GB FILE SIZES
+     * @param file
+     *            - The File to upload
+     * @param dropZone
+     *            - The target dropzone 
+     * @return - Boolean, success
+     */
+    public boolean upLoadMiniAppFile(File uploadFile, DropZone dropZone)
```

Comment:
Can you please change this to uploadMiniAppFile() so it fits the same camel case that we use for uploadFile?